File: FoodN_WebService/FoodN_WebService/Store.cs
using System;
using System.Collections.Generic;
using System.Data.Linq;
using System.Linq;
using System.Web;

namespace FoodN_WebService
{
    public class Store
    {
        string tenCuaHang;

        public string TenCuaHang
        {
            get { return tenCuaHang; }
            set { tenCuaHang = value; }
        }
        string diaChi;

        public string DiaChi
        {
            get { return diaChi; }
            set { diaChi = value; }
        }
        string gioDongCua;

        public string GioDongCua
        {
            get { return gioDongCua; }
            set { gioDongCua = value; }
        }
        string gioMoCua;

        public string GioMoCua
        {
            get { return gioMoCua; }
            set { gioMoCua = value; }
        }
        Byte[] hinhAnh;

        public Byte[] HinhAnh
        {
            get { return hinhAnh; }
            set { hinhAnh = value; }
        }
        string sdtCuaHang;

        public string SdtCuaHang
        {
            get { return sdtCuaHang; }
            set { sdtCuaHang = value; }
        }
    }
}

File: FoodNtest/app/src/main/java/com/example/admin/model/Store.java
package com.example.admin.model;

import android.graphics.Bitmap;

import com.google.android.gms.maps.model.Marker;

import java.util.ArrayList;

/**
 * Created by Windows 10 on 05/05/2018.
 */

public class Store {
    public Integer getID() {
        return ID;
    }

    public void setID(Integer ID) {
        this.ID = ID;
    }

    private Integer ID;
    private String tenCuaHang;
    private String diaChi;
    private String gioMoCua;
    private String gioDongCua;
    private Bitmap hinhAnh;
    private String sdtCuaHang;
    private Marker marker;

    public Marker getMarker() {
        return marker;
    }

    public void setMarker(Marker marker) {
        this.marker = marker;
    }

    public ArrayList<Position> getListPoint() {
        return listPoint;
    }

    public void setListPoint(ArrayList<Position> listPoint) {
        this.listPoint = listPoint;
    }

    private ArrayList<Position> listPoint = new ArrayList<>();

    public String getTenCuaHang() {
        return tenCuaHang;
    }

    public void setTenCuaHang(String tenCuaHang) {
        this.tenCuaHang = tenCuaHang;
    }

    public String getDiaChi() {
        return diaChi;
    }

    public void setDiaChi(String diaChi) {
        this.diaChi = diaChi;
    }

    public String getGioMoCua() {
        return gioMoCua;
    }

    public void setGioMoCua(String gioMoCua) {
        this.gioMoCua = gioMoCua;
    }

    public String getGioDongCua() {
        return gioDongCua;
    }

    public void setGioDongCua(String gioDongCua) {
        this.gioDongCua = gioDongCua;
    }

    public Bitmap getHinhAnh() {
        return hinhAnh;
    }

    public void setHinhAnh(Bitmap hinhAnh) {
        this.hinhAnh = hinhAnh;
    }

    public String getSdtCuaHang() {
        return sdtCuaHang;
    }

    public void setSdtCuaHang(String sdtCuaHang) {
        this.sdtCuaHang = sdtCuaHang;
    }
}
